### DISCUSSION TITLE
Error while installing foundry

### DISCUSSION THREAD

**Discussion by AdityaLohar** [foundry-rs/foundry#6167] [Q&A]
I have been trying to install foundry but getting the error " **curl: (60) SSL certificate problem: unable to get local issuer certificate** ".
I have tried to update the certificates but it did not help. Please help !

![Screenshot 2023-10-30 131132](https://github.com/foundry-rs/foundry/assets/77808500/979a9806-cbc5-41f5-9d53-86ae569ecabb)

**Comment by temisan0x** [comment]
Hello @AdityaLohar have you been able to resolve this issue?
